src/styles/blocks/header/desktop-nav.scss

```
@@ -0,0 +1,34 @@
+.desktop-nav {
+  display: none;
+  max-width: 1400px;
+  margin: 0 auto;
```

```suggestion
  margin: 0 auto;
```
You can not use outer spacings (margins) for BEM-blocks
Now you already have block `nav` so you can create elements `nav__desktop` and `nav__mobile`